### ISSUE TITLE
[BUG] div tags following long attributes are not indented correctly

### ISSUE THREAD

**Issue by christopherpickering** [Riverside-Healthcare/djLint#31]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: e.g. ubuntu 20.04
 - Python Version (``python --version``)
 - djLint Version (``djlint --version``)

## Issue

```html
{% block body %}
    <div class="em-ajaxTable"
         data-theme="Exec"
         data-src="/table/project/{% if user_id %}{{ user_id }}{% elif mine %}mine{% else %}all{% endif %}">
        <div class="loader"></div></div>
    {% endblock body %}
```
shows as correct html, but the ``</div>`` tags should be on newlines.

**Comment by codetoadsnow** [comment]
Hi. Did this bug correct? I still have it.

**Comment by christopherpickering** [comment]
Hi, Can you open a new bug with some example code?
